src/components/NonProfitApply/NonProfitApply.js

```diff
@@ -143,10 +141,12 @@ export default function NonProfitApply() {
 
     fetch("http://localhost:3001/submit-application", options)
       .then((response) => {
-        return response.json();
+        if (response.ok) {
+          router.push("http://localhost:3000/nonprofits/confirmation");
```

Comment:
@lizanaya92 - hardcoded localhost here, should be `/nonprofits/confirmation` - I think